Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -4102,7 +4102,8 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         new DiscretePropertyWrapper<const AtomString&>(CSSPropertyPositionAnchor, &RenderStyle::positionAnchor, &RenderStyle::setPositionAnchor),
         new DiscretePropertyWrapper<const BlockEllipsis&>(CSSPropertyBlockEllipsis, &RenderStyle::blockEllipsis, &RenderStyle::setBlockEllipsis),
         new DiscretePropertyWrapper<size_t>(CSSPropertyMaxLines, &RenderStyle::maxLines, &RenderStyle::setMaxLines),
-        new DiscretePropertyWrapper<OverflowContinue>(CSSPropertyContinue, &RenderStyle::overflowContinue, &RenderStyle::setOverflowContinue)
+        new DiscretePropertyWrapper<OverflowContinue>(CSSPropertyContinue, &RenderStyle::overflowContinue, &RenderStyle::setOverflowContinue),
+        new DiscretePropertyWrapper<const TimelineScope&>(CSSPropertyTimelineScope, &RenderStyle::timelineScope, &RenderStyle::setTimelineScope),
```

Comment:
Spec says this is not animatable, so it should not have a wrapper: https://drafts.csswg.org/scroll-animations/#propdef-timeline-scope

Reply:
Fixed.